examples/validate/check_whitelist_registries.yaml

```diff
@@ -17,5 +17,5 @@ spec:
             spec:
               containers:
               - name: "*"
-                # Checks if the image path starts with "https://hub.docker.io/nirmata/*"
-                image: https://hub.docker.io/nirmata/*
+                # Check allowed registries
+                image: "*/nirmata/* | https://private.registry.io/*"
```

Comment:
There should be no spaces around the | character, because those spaces will be processed as a part of values

Reply:
@belyshevdenis - what was the decision on this?